summit/2018.html

```diff
@@ -54,20 +54,36 @@ <h2 class="text">Schedule</h2>
         </div>
       </div>
 
-      <div class="row">
+    <div class="row">
       <div class="col-md-10 col-md-offset-1">
         <img class="sponsor" src="/images/summit/megaphone.svg" width="65px" height="auto" alt="Sponsor"/>
-        <h2 class="text">Supporters</h2>
+        <h2 class="text">Sponsors</h2>
        <a href="https://stickermule.com/" target="_blank" rel="noopener"><img class="sponsor" src="/images/summit/stickermule-logo.svg" width="300px" height="auto" alt="Stickermule"/>
         <p class="data">Stickermule</p></a>
         <a href="https://mozilla.org/" target="_blank" rel="noopener"><img class="sponsor" src="/images/summit/mozilla-logo.svg" width="250px" height="auto" alt="Mozilla"/>
         <p class="data">Mozilla</p></a>
-        <a href="https://opentech.fund/" target="_blank" rel="noopener"><img class="sponsor" src="/images/summit/opentechfund-logo.svg" width="250px" height="auto" alt="OpenTechFund"/>
-        <p class="data">OpenTechFund</p></a>
         <a href="https://phplist.org/" target="_blank" rel="noopener"><img class="sponsor" src="/images/summit/phplist-logo.svg" width="220px" height="auto" alt="phpList"/>
         <p class="data">phpList</p></a>
       </div>
     </div>
+
+    <div class="row">
+      <div class="col-md-10 col-md-offset-1">
+        <img class="supporter" src="/images/summit/supporters.svg" width="65px" height="auto" alt="Sponsor"/>
```

Comment:
alt=Supporter

Reply:
Actually should be `alt=""` (empty) because the image is purely decoration, as the h2 has "Supporters" already. :)